### ISSUE TITLE
NPE for null data values

### ISSUE THREAD

**Issue by chrisgleissner** [MitchTalmadge/ASCIIGraph#11]
An NPE is thrown when some of the data cells are null. It would be good if these could be rendered in a special way (e.g. as "null" or configurable via a new method `withNullValue(String)` ) to differentiate them from empty strings.

Example to reproduce the NPE:

```
System.out.println(ASCIITable.fromData(new String[]{"ID", "Name"}, new String[][]{{"1", null},}).toString());
```

This results in:
```
Exception in thread "main" java.lang.NullPointerException
	at com.mitchtalmadge.asciidata.table.ASCIITable.<init>(ASCIITable.java:79)
```
I appreciate that I could convert the data before passing it into ASCIITable, but having absent values is quite a common scenario and I think your library would be improved by handling this gracefully.

**Comment by MitchTalmadge** [comment]
You're right, this is quite a simple fix. I will get this out when I can. Until then, I encourage you to convert the null values on your side. Thank you for using the library and thank you for the report!

**Comment by chrisgleissner** [comment]
Hi Mitch,

Thanks for your reply. If I fix it as part of a merge request, would you
consider merging it? I wanted to check first before I start work on it.

Thanks
Christian

**Comment by MitchTalmadge** [comment]
Certainly I would, and this would help me too with my busy schedule. Thank you!

**Comment by chrisgleissner** [comment]
Great. I already implemented it and will raise a pull request this evening.

**Comment by chrisgleissner** [comment]
I raised the PR. My local build passes, but the PR's automatic build fails due to a key issue. The changes were quite simple. It would be great if you could have a look and cut a new Maven Repository release if you like them as I need the changes for a project I am working on. Thanks

https://travis-ci.org/MitchTalmadge/ASCII-Data/builds/450411245

**Comment by MitchTalmadge** [comment]
Thank you! The error is because you committed to master instead of develop (my bad, I didn't warn you) and so Travis has some protections in place to prevent decryption on external PRs. It's not a problem though and I will merge the request.

**Comment by MitchTalmadge** [comment]
Released as 1.3.0. Let me know how it goes. Thank you for the quality code and I really appreciated that you wrote test cases. :)

**Comment by chrisgleissner** [comment]
Hi Mitch,

Thanks, that makes perfect sense. I had been wondering about your dev
branch. Thanks for merging.

Best wishes
Christian
